suites/pacific/rados/sanity_rados.yaml

```diff
@@ -49,6 +49,23 @@ tests:
       destroy-cluster: false
       abort-on-fail: true
 
+  - test:
+      name: Testbed setup for RADOS
+      module: rados_prep.py
+      config:
+        ec_pool: true
+        replicated_pool: true
+        email_alerts:
+          smtp_host: smtp.corp.redhat.com
+          smtp_sender: ceph-iad2-c01-lab.mgr@redhat.com
+          smtp_port: 25
+          interval: 10
+          smtp_destination:
+            - pdhiran@redhat.com
```

Comment:
We can use a distribution list instead of a member's email address. But there are email alerts generated which will be sent to entire DL, which might not be something everyone wants to receive. I agree with not using personal mail ID's. We can decide on destination email address.